src/files/inheritance.ts

```diff
@@ -50,7 +50,7 @@ export function* walkBack<T extends string[]>(
     if (candidates.length > 1) {
       const exactMatch = candidates.find((file) => {
         const { entities } = readEntities(file.name)
-        return Object.keys(sourceParts.entities).every((entity) =>
+        return [...Object.keys(sourceParts.entities), ...(targetEntities ?? [])].every((entity) =>
```

Comment:
This is a logic bug that doesn't matter in the current operations because we've only cared to get the first value, but what was happening was that an `ieeg.*` file with multiple `space-*_electrodes.tsv` files would just match the first one because every entity in `ieeg` was also in `electrodes`, and checking that `entities['space'] === sourceEntities['space']` wasn't happening.

It should be fixed and tested in a separate PR, but the bug needed fixing to make this work.